### DISCUSSION TITLE
世界第一座「市中心機場」，專停飛行器和無人機

### DISCUSSION THREAD

**Discussion by Lai27330** [1101-GS4719-C/Python-Community-GS4719-C#588] [General]
無人機機場體積小不占空間，又加上大大降低了噪音和汙染的問題，除去經常聽到的無人機燈光表演和攝影，對於航空運輸物資及偏遠地區的救災也將產生更大的貢獻，但雖然無人機方便快捷、能適應多種環境、功能多樣並且環保低污染，聽起來好處多多，但是關於無人機與航空法的衝突像是多起闖入民航機場起飛範圍影響飛行的事件或因操作不當危及地面人員安全的部分，也應該制定相關嚴謹的法規來應對，而對於訊號干擾產生的威脅，如何排除也是一項值得深入研究的議題。
以下附上網址 :
https://technews.tw/2022/05/03/urban-air-port_air-one-drone-landing-pad/

**Comment by Ku-Hao-Chen** [comment]
謝謝分享，希望未來可以普及。

**Comment by HsuChen88** [comment]
未來如果用無人機來運送包裹，再加上自動化的流程，就能省下時間與人力了。

**Comment by Jay-722** [comment]
無人機的應用已經越發蓬勃，不管什麼產業都會有能結合的地方，伴隨產生的問題值得關注

**Comment by 11timyyy** [comment]
在使用無人機時，也要密切注意周圍的環境

**Comment by handsomeniko** [comment]
無人機也是未來趨勢

**Comment by Sazmafuji** [comment]
希望此技術，不要用於戰爭

**Comment by cordydeng** [comment]
無人機雖然還不是主流，但相信未來無人機的市場一定會後勢看漲

**Comment by gnsJhenJie** [comment]
不知道未來這種低空飛行的無人機普及的時候，會不會像飛機一樣有交通管制的問題。

**Comment by henrywen547** [comment]
空運能解決很多陸運無法克服的環境，隨著無人機AI發展以及體積逐漸縮小，能協助人類降低成本並解決更多問題

**Comment by Rosancl** [comment]
在市中心蓋無人機機場是新實踐的事，實行的好或許有機會成為未來趨勢，但畢竟在市中心，可能還是得考慮假設機械故障導致意外降落馬路是否會釀成其他交通問題。

**Comment by jason79461385** [comment]
無人機用來運送物資等真的很方便，若將來普及的話，對於物流業者應該會是一大福音。

**Comment by sys511613** [comment]
無人機現在的發展已經越來越成熟
在各個領域上也都漸漸可以看到無人機的出現
但最期待的還是希望早點能運用在物流上
這樣網路購物會變得更加方便
不用再看貨物卡在物流中心好幾天不動了

**Comment by yucheng1028** [comment]
感覺貨物安全會是很大的問題 美國連火車貨物都可以被搶成那樣了 無人機可想而知
